test/google_service_account_test.go

```diff
@@ -0,0 +1,105 @@
+package test
+
+import (
+	"github.com/gruntwork-io/terratest/modules/logger"
```

Comment:
go import

Reply:
use command goimports to sort your imports
https://pkg.go.dev/golang.org/x/tools/cmd/goimports

Reply:
done